docs/scos/dev/front-end-development/migration-guide-upgrade-to-webpack-v5.md

```diff
@@ -0,0 +1,263 @@
+---
+title: Migration guide - Upgrade to Webpack v5
+description: Use the guide to update versions of the Webpack and related modules.
+template: module-migration-guide-template
+---
+
+## Upgrading from version 4.* to version 5.*
+
+This document provides instructions for upgrading Webpack to version 5 in your Spryker project.
+
+## Overview
+
+The Webpack v5 was released 2 years ago.
+So we need to migrate as well to unblock future upgrades of other dependencies.
+
+*Estimated migration time: 1h*
+
+## 1) Update dependencies
+
+{% info_block infoBox "Note" %}
+
+To ensure that dependencies will install correctly, delete `node_modules` and `package-lock.json` before installation 
+using the next command `rm -rf node_modules && rm -rf package-lock.json`.
+
+{% endinfo_block %}
+
+1. Update the following dependencies to the new version:
+
+```json
+{
+    // update
+    "@types/webpack": "~5.28.0", // If using Merchant Portal
+    "clean-webpack-plugin": "~4.0.0",
+    "compression-webpack-plugin": "~10.0.0",
+    "copy-webpack-plugin": "~11.0.0",
+    "mini-css-extract-plugin": "~2.7.2",
+    "optimize-css-assets-webpack-plugin": "~6.0.1",
+    "terser-webpack-plugin": "~5.3.6",
+    "webpack": "~5.74.0",
+    "webpack-merge": "~5.8.0",
+    // add
+    "postcss": "~8.4.20"
```

Comment:
But I don't see the dependency in the [PR](https://github.com/spryker/suite-nonsplit/pull/7449)

Reply:
`yves:production` requires postCSS version 8

Reply:
OK, but why don't we have this in our [package.json](https://github.com/spryker/suite-nonsplit/blob/a3d1e41973c6236a5dd37e9e03b185ddc4bfd251/package.json)?

Reply:
Seems some other dependency covers it

Reply:
So I think we should add it to avoid an unexpected issue.

Reply:
Added